test/pki-secret-engine-certificate-sample.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: cert-manager.io/v1
```

Comment:
do you have documented the test sequence to run the manual test with cert-manager? I have not seen it.
There should be a readme in the test folder.

Reply:
I will document it

Reply:
ok, shall I move it in a dedicated directory for PKI engine with the readme.md?